Improved automation by adding -accepteula option

cherokeejb opened the pull request:
**Details:**
added -accepteula option to 
1) psexec:
    on atomic-red-team/atomics/T1569.002/T1569.002.yaml
2)psexec also on  T1021 test 3
3) psexec also on T1021.006
4) autoruns: reviewed, is only used to attempt copy in ADS there and not executed
   on atomics/T1564.004/T1564.004.yaml, no change

+)Added line that type of commands be highlighted in the spec.yaml instructional document.

**Testing:**
Only added command flags, did not test

**Associated Issues:**
N/A

cnotin commented on atomic_red_team/spec.yaml:
```diff
@@ -92,7 +95,8 @@ atomic_tests:
       where sysmon > nul
     get_prereq_command: |
-      echo Automated installer not implemented yet, please install Sysmon manually
+      echo Automated installer not implemented yet, please install Sysmon 
+      ly
```
I think this was unintentional ;)

cnotin commented on atomic_red_team/spec.yaml:
```diff
@@ -20,6 +20,9 @@
 # Use git symlinks if you really need to share files between techniques.
 #
+# Atomic tests should be fully automated whenever possible, requiring no continued interaction.
+# Include any needed options to execute the commands seamlessly, for exapmle SysInternal's -accepteula option or any -q or -quiet modes.
```
```suggestion
# Include any needed options to execute the commands seamlessly, for example SysInternal's -accepteula option or any -q or -quiet modes.
```

cnotin commented:
Also, I think you forgot T1021.006 which you mentioned in https://github.com/redcanaryco/atomic-red-team/issues/1092#issuecomment-660598611

cherokeejb commented:
> Thanks JB. Does esentutl really have an accepteula parameter? I don't see a reference to one on the MS documentation but maybe I missed it. Also please add "manually" back in as noted by cnotin

Hi Carrie! - The -accepteula that I added wasn't for esentutl -- that's not part of sysinternals, but it might also have a similar command-- it was on the 'autoruns' command.  I've fixed the word 'manually' above, like Clément noted, in my patch31 so it should merge correctly when this is ready.

cherokeejb commented:
@clr2of8  just for handy reference, so at the end of [https://github.com/redcanaryco/atomic-red-team/blob/24549e3866407c3080b95b6afebf78e8acd23352/atomics/T1564.004/T1564.004.yaml] I believe what is going on is they are using esentutl to add commands into alternate data streams - for example, so that would be preventing a future hang-up from autoruns 

from line 30:
 #{path}\file.txt:autoruns.exe /o   <--- suggested add '/accepteula' here
(c:\ADS\\file.txt:autoruns.exe /o )

cherokeejb commented:
@clr2of8 yes I think the esentutl line was all part of esentutl, just embedding the post-compression autoruns in an ADS (so not executing a sysinternals tool there).
